scripts/microbe_stage/microbe_editor/microbe_editor.as

```diff
@@ -227,7 +227,11 @@ class MicrobeEditor{
 
     private void _addOrganelle(PlacedOrganelle@ organelle)
     {
-        EditorAction@ action = EditorAction(organelle.organelle.mpCost,
+		if((organelle.organelle.name == "nucleus" && !nucleusIsPresent) || organelle.organelle.name != "nucleus")
+		{
+			if(organelle.organelle.name == "nucleus")
```

Comment:
There are tabs here. Run the formatting script.

Reply:
actyally in my code there are 4 space.. i don't know why when i make push and then pull they change

Reply:
FAILED to format file: ./ThirdParty/Leviathan/build/ThirdParty/include/bullet/VHACD/test/inc/oclHelper.h

Now when i lunch formatting code i get this, feels bad